ckanext/dashboard/templates/package/read.html

```diff
@@ -0,0 +1,14 @@
+{% extends "package/read.html" %}
+
+{% block package_description %}
+  {{ super() }}
+  
+  {# Llamamos al helper para obtener la configuración del dashboard #}
+  {% set dashboard = h.get_dataset_dashboard(package['id']) %}
```

Comment:
```suggestion
  {% set dashboard = h.get_dataset_dashboards(package['id']) %}
```

Should we allow and display multiple dashboards for a single package?